python/lsst/faro/measurement/VisitTableMeasurement.py

```diff
@@ -0,0 +1,48 @@
+import lsst.pipe.base as pipeBase
+from lsst.verify.tasks import MetricConnections
+import lsst.pex.config as pexConfig
+
+from lsst.faro.base.CatalogMeasurementBase import CatalogMeasurementBaseTaskConfig, CatalogMeasurementBaseTask
+
+__all__ = ("VisitTableMeasurementTaskConfig", "VisitTableMeasurementTask")
+
+
+class VisitTableMeasurementTaskConnections(MetricConnections,
+                                           dimensions=("instrument", "visit", "band")):
+
+    catalog = pipeBase.connectionTypes.Input(doc="Source catalog for visit.",
+                                             dimensions=("instrument", "visit", "band"),
+                                             storageClass="DataFrame",
+                                             name="sourceTable_visit",
+                                             deferLoad=True)
+
+    measurement = pipeBase.connectionTypes.Output(doc="Per-visit measurement.",
+                                                  dimensions=("instrument", "visit", "band"),
+                                                  storageClass="MetricValue",
+                                                  name="metricvalue_{package}_{metric}")
+
+
+class VisitTableMeasurementTaskConfig(CatalogMeasurementBaseTaskConfig,
+                                      pipelineConnections=VisitTableMeasurementTaskConnections):
+    columns = pexConfig.Field(doc="Columns from sourceTable_visit to load.",
+                              dtype=str, default='coord_ra, coord_dec')
+
+
+class VisitTableMeasurementTask(CatalogMeasurementBaseTask):
+    ConfigClass = VisitTableMeasurementTaskConfig
+    _DefaultName = "visitTableMeasurementTask"
+
+    def run(self, catalog, dataIds):
+        return self.measure.run(catalog, self.config.connections.metric, dataIds)
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        columns = [_.strip() for _ in self.config.columns.split(',')]
+        inputs['catalog'] = inputs['catalog'].get(parameters={'columns': columns})
```

Comment:
I'm a little confused by this.  I think `butlerQC.get` should have returned a `pandas.DataFrame` already, and the get method on a `DataFrame` does not take a parameters argument.  Did you perhaps mean to pop off the catalog entry from `inputRefs` and use that to get the catalog and put it in `inputs['catalog']`?

Reply:
The objective here was to selectively load a subset of columns from the parquet table. The input for the `sourceTable_visit` uses a deferred load so that one can use `get` her with the `parameters` argument specified. Is there another way to do this?

Reply:
The problem is that you are replacing a catalog reference with a data frame. You should use different variable names.

Reply:
Now using a different variable name, `catalog` instead of `inputs['catalog']` so that `inputs['catalog']` is not modified.